### PR TITLE
[Audit] Add option to check last digits in Benfords Law

### PR DESCRIPTION
**Pull request by koenderks**
Implements https://github.com/jasp-stats/jasp-issues/issues/815

Accompanies https://github.com/jasp-stats/Audit/pull/9 to the Audit module

### CODE REVIEW

File: JASP-Engine/JASP/R/auditClassicalBenfordsLaw.R [hunk 1]
```diff
@@ -182,14 +186,20 @@ auditClassicalBenfordsLaw <- function(jaspResults, dataset, options, ...){
       leadingDigits <- table(as.numeric(substring(format(abs(obs), scientific = TRUE), 1, 3)) * 10)
       digits <- 10:99
 
+    } else if(options[["digits"]] == "last"){
+
+      stringedObs <- format(abs(obs), scientific = TRUE)
+      leadingDigits <- table(as.numeric(substring(stringedObs, nchar(stringedObs), nchar(stringedObs))))
+      digits <- 1:9
```
Comment (JorisGoosen) [inline JASP-Engine/JASP/R/auditClassicalBenfordsLaw.R:193]: Aren't you missing 0?

Comment (JorisGoosen) [reply JASP-Engine/JASP/R/auditClassicalBenfordsLaw.R:193]: As in, something can end in "0.12300231" right?

Comment (koenderks) [reply JASP-Engine/JASP/R/auditClassicalBenfordsLaw.R:193]: Yes, but 0 doesn't have a probability according to Benford's Law.

Comment (AlexanderLyNL) [reply JASP-Engine/JASP/R/auditClassicalBenfordsLaw.R:193]: Have you tried this with values such as 5.60? What should the procedure do then? Skip 0 the zero and check 6?

**Pull request merged**